pkgs/tools/misc/reredirect/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "reredirect";
+  version = "v0.2";
+
+  src = fetchFromGitHub {
+    owner = "jerome-pouiller";
+    repo = "reredirect";
+    rev = version;
+    sha256 = "0aqzs940kwvw80lhkszx8spcdh9ilsx5ncl9vnp611hwlryfw7kk";
+  };
+
+  makeFlags = [ "PREFIX=$(out)" ];
```

Comment:
This was required?

Reply:
Have a look: https://github.com/jerome-pouiller/reredirect/blob/master/Makefile

Reply:
Yup. Thanks. In that case `makeFlags = [ "PREFIX=${placeholder "out"}" ];` please.

Reply:
Great catch. Fixed.